x-pack/plugins/session_view/public/components/detail_panel_metadata_tab/helpers.test.ts

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
Not a huge deal, but for next time best to use 'git mv <file> <new_file>' when renaming files, this will ensure git history is maintained across the rename, and will make review easier.